docs/docs/firmware/touch-panel-esp32.md

```diff
@@ -2400,6 +2402,27 @@ Custom icons are **not persistent** and have to be (re)loaded after restart and
 * Icons are aligned top/left of a tile
 * There are no size checks for custom icons
 
+## Firmware Installation
+
+### Initial Set-up – Windows 10
+Workflow to commission a new Touch Panel:
+1.	[Download](https://github.com/OXRS-IO/OXRS-IO-TouchPanel-ESP32-FW/releases) the most recent wifi or ethernet version of the compiled *oxrs-io-touchpanel-esp32-fw* firmware to a work folder created for the commissioning process.

```

Comment:
Could we match the case of the release binary files generated - i.e. `OXRS-IO-TouchPanel-ESP32-FW.*`?